scripts/hudson/narayana.sh

```diff
@@ -189,7 +189,7 @@ function init_test_options {
     elif [[ $PROFILE == "LRA" ]]; then
         if [[ ! $PULL_DESCRIPTION_BODY == *!LRA* ]]; then
           comment_on_pull "Started testing this pull request with LRA profile: $BUILD_URL"
-          export AS_BUILD=0 AS_CLONE=0 AS_DOWNLOAD=1 AS_TESTS=0 NARAYANA_BUILD=0 NARAYANA_TESTS=0 XTS_AS_TESTS=0 XTS_TESTS=0 TXF_TESTS=0 txbridge=0
+          export AS_BUILD=0 AS_CLONE=0 AS_DOWNLOAD=1 AS_TESTS=0 NARAYANA_BUILD=1 NARAYANA_TESTS=0 XTS_AS_TESTS=0 XTS_TESTS=0 TXF_TESTS=0 txbridge=0
```

Comment:
If you want to use a cloned and build version of the AS you could change `AS_BUILD=0 AS_CLONE=0 AS_DOWNLOAD=1` to `AS_BUILD=1 AS_CLONE=1 AS_DOWNLOAD=0`

Reply:
Okay but my script changes makes the `AS_CLONE` variable redundant since I always build WildFly if the build needs to test against WildFly.

Reply:
I think reconfiguring the PROFILE here would be clearer

Reply:
I'm not aware of any scenario where one would want to clone the AS and not build it.
But if you require it then I will look into re-instating AS_CLONE.

Reply:
What I am suggesting at this particular line is to change "AS_BUILD=0 AS_CLONE=0 AS_DOWNLOAD=1" to "AS_BUILD=1 AS_CLONE=1 AS_DOWNLOAD=0" (and to keep your change "NARAYANA_BUILD=1")